### ISSUE TITLE
Can't find .css files with Font Awesome 6 Free

### ISSUE THREAD

**Issue by z1glr** [sebinside/StreamAwesome#248]
I'm (still) experiencing a similar problem as in https://github.com/sebinside/StreamAwesome/issues/207 
> [plugin:vite:import-analysis] Failed to resolve import "../fonts/fontawesome/css/sharp-light.min.css" from "src/main.ts". Does the file exist?

Somehow the checks implemented by https://github.com/sebinside/StreamAwesome/pull/208 don't seem to work. I tried changing the `fontAwesomeVersionInfo` object in `model/versions.ts` to
```ts
fontAwesomeVersionInfo = {
  fontFamilyBase: 'Font Awesome 6',
  fontVersion: '6.6.0',
  fontLicense: 'Free'
}
```
but this doesn't help either. Replacing the if-statement in `main.ts` with
```ts
...
if (false) {
...
```
does work however.

## Reproduce
1. Delete the fonts/fontawesome folder
1. Download font awesome free for web [Download](https://fontawesome.com/download)
1. Paste in the folders as stated in the README
1. npm run dev

**Comment by sebinside** [comment]
Yes, @DerArkeN with #208 fixed this but caused the final non-debug build to fail as the files are included dynamically and thus not considered by the build. There are ways to statically force folders to be included in the build, however, I was not able to get it working prior to releasing v3, so I partially reverted it.

Nevertheless, I am still interested in a version that defaults to the Free version if the pro files are not included. So everyone is invited to have look at it. I failed in, e.g., altering the vite config file to include the folders no matter what, when running `npm run build`.

**Comment by sebinside** [comment]
Fixed by #287.
